rust_src/remacs-sys/lib.rs

```diff
@@ -23,6 +23,7 @@ use libc::{c_char, c_uchar, c_short, c_int, c_double, c_float, c_void, ptrdiff_t
 
 
 include!(concat!(env!("OUT_DIR"), "/definitions.rs"));
+include!(concat!(env!("OUT_DIR"), "/globals.rs"));
```

Comment:
The comment on line 6 should probably mention `emacs_globals` too, unless I'm missing something.